src/components/inventory/sidebar.tsx

```
@@ -0,0 +1,85 @@
+
+import React, { useEffect, useState } from "react";
+import { AwaitedPageProps } from "@/config/types";
+import { useRouter } from "next/navigation";
+import { parseAsString, useQueryStates } from "nuqs";
+import { color } from "motion/react";
+import { routes } from "@/config/routes";
+import { env } from "@/env";
+import { cn } from "@/lib/utils";
+
+interface SidebarProps extends AwaitedPageProps {
+  minMaxValues: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)